commands.json

```diff
@@ -1,40 +1,41 @@
 {
   "FT.CREATE": {
     "summary": "Creates an index with the given spec",
-    "complexity": 
-      "O(K) at creation where K is the number of fields, O(N) if scanning the keyspace is triggered, where N is the number of keys in the keyspace",
-      "history": [        
-        [
-          "2.0.0",
-          "Added `PAYLOAD_FIELD` argument for backward support of `FT.SEARCH` deprecated `WITHPAYLOADS` argument"
-        ],
-        [
-          "2.0.0",
-          "Deprecated `PAYLOAD_FIELD` argument"
-        ]
+    "complexity": "O(K) at creation where K is the number of fields, O(N) if scanning the keyspace is triggered, where N is the number of keys in the keyspace",
+    "history": [
+      [
+        "2.0.0",
+        "Added `PAYLOAD_FIELD` argument for backward support of `FT.SEARCH` deprecated `WITHPAYLOADS` argument"
       ],
+      [
+        "2.0.0",
+        "Deprecated `PAYLOAD_FIELD` argument"
+      ]
+    ],
     "arguments": [
       {
         "name": "index",
-        "type": "string"
+        "type": "string",
+        "summary": "Specifies the name of the index. The index must be created using `FT.CREATE`."
```

Comment:
Not needed? (this an argument of `FT.CREATE`)
> The index must be created using `FT.CREATE`.

Reply:
Changed to:
"Specifies the name of the index."